CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 See the [releases](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases) for details on bug fixes and added features.
 
+7.2.0
+=====
+### Fundamentals
+Improve logging in Wilson for failed token validation when key not found [#2436](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2436)
```

Comment:
I think this needs to be added: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/f7ccbb409c078f92e81d950fe256c1db54daad06

and this https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/f40dd9029c57167ce836bb4100a33dd5e1bf7d09

and maybe these under fundamentals:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/e25592ae38350e32a192255b05ffc43abc8acec6

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/21d2f6d7e69161f1f9b08d6b21342703bcd33354

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/8cc08dcb45284b9b4fcf3de361095151a4ef800f

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/4189db309d2793461af181bf1d51bdb6c6b4c881

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/c1be3d6d518fdd01195b0c682bbc3d189dd1286f

but can give a better name and link to the PR instead.

Reply:
These have all been added